src/Components/Artwork/GridItem.tsx

```diff
@@ -167,7 +202,28 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
               }
             }}
           >
-            <Image src={this.getImageUrl()} />
+            {preloadImage ? (
+              <Image
+                title={artwork.title}
+                alt={artwork.image_title}
+                src={this.getImageUrl()}
+              />
+            ) : (
+              <LazyImage
+                title={artwork.title}
+                alt={artwork.image_title}
+                style={{ opacity: isImageLoaded ? "1" : "0" }}
+                src={this.getImageUrl()}
+                onLoad={this.imageLoaded.bind(this)}
+              />
+            )}
+            <noscript>
+              <Image
+                title={artwork.title}
+                alt={artwork.image_title}
+                src={this.getImageUrl()}
+              />
+            </noscript>
```

Comment:
SEO?

Reply:
Neither. Just the no js use case. I mean, it _might_ be unnecessary... so much of our site wouldn't work without js anyway. But.. yeah. SEO effectively ignores this and I doubt it'd be used for accessibility... though I'm not 100% sure of that.

Reply:
OH duh, for no JS -- but that said, our new pages are totally browsable / usable without JS turned on so we want to keep things fluid where possible -- this seems like a good solution 👍 .